sound/soc/sof/intel/hda-stream.c

```diff
@@ -290,13 +291,21 @@ int hda_dsp_stream_put(struct snd_sof_dev *sdev, int direction, int stream_tag)
 			continue;
 
 		if (s->direction == direction && s->stream_tag == stream_tag) {
+			mask = BIT(hext_stream->hstream.index);
```

Comment:
Should it be `|=`? Can we assume that there is only one stream found?

Reply:
It should be `=` we cannot have multiple streams with the same stream_tag